src/lib/messenger.js

```diff
@@ -23,6 +23,19 @@ export default function messenger(bot) {
       const text = message.text.toLowerCase();
       const user = message.user || {};
 
+      function stripText(input) {
```

Comment:
Yeah, I've thought about it myself, maybe cleanText?
Or something like that?
Because I want it to contain the word text for association with the constant.